indico/modules/events/registration/models/registrations.py

```diff
@@ -226,6 +226,11 @@ class Registration(db.Model):
             lazy=True
         )
     )
+    #: The registration tags assigned to this registration
+    registration_tags = db.relationship(
+        'RegistrationRegistrationTag',
+        lazy='dynamic'
```

Comment:
I don't think this is a good option here

Reply:
you may want to make this a proper many2many relationship (`secondary` attribute pointing to the mapping table, which would not be a full-blown `db.Model` subclass)